index.html

```diff
@@ -162,17 +162,17 @@ <h3 class="fs-600 text-primary-400 fw-bold">
               </div>
               <div class="padding-block-900">
                 <ul role="list" class="numbered-list padding-inline-400">
-                  <li style="display: flex; align-items:center">
+                  <li style="display: flex; align-items: center">
                     <div
-                    class="text-priamry-400 fs-primary-heading fw-semi-bold flow"
-                  >
-                    1
-                  </div>
-                  <h4 class="fs-400 fw-semi-bold padding-block-400">
-                    Full service range including
-                  </h4>
+                      class="text-priamry-400 fs-primary-heading fw-semi-bold flow"
+                    >
+                      1
+                    </div>
+                    <h4 class="fs-400 fw-semi-bold padding-block-400">
+                      Full service range including
+                    </h4>
                   </li>
-                  <li style="display: flex; align-items:center">
```

Comment:
wszędzie gdzie masz zastosowany styl w HTML -> usuwamy to. Nie powinno się mieszać styli z plikami HTML bo bardzo łatwo o nieporządek i późniejsze problemy przy rozwijaniu kodu.